data/card_catalogue/movement.json

```diff
@@ -126,7 +126,7 @@
         "passive": null
```

Comment:
Oh I think I figured out why these don't work, they still say active and passive

Reply:
I'm embarrassed by how long I spent trying to figure out what was wrong with these lol, fixed in https://github.com/elliottomlinson/cardmaster/pull/214/commits/98ce1a99d278e575e166f9636aaeac8bc2a0c0ad